content/docs/started/getting-started-k8s.md

```diff
@@ -144,11 +144,342 @@ kubectl get cm dex -n kubeflow -o jsonpath='{.data.config\.yaml}' > dex-config.y
+                  ports:
+                    - containerPort: 389
+                      name: openldap
+                  env:
+                    - name: LDAP_LOG_LEVEL
+                      value: "256"
+                    - name: LDAP_ORGANISATION
+                      value: "Arrikto"
+                    - name: LDAP_DOMAIN
+                      value: "arrikto.com"
+                    - name: LDAP_ADMIN_PASSWORD
+                      value: "admin"
+                    - name: LDAP_CONFIG_PASSWORD
+                      value: "config"
+                    - name: LDAP_BACKEND
+                      value: "mdb"
+                    - name: LDAP_TLS
+                      value: "false"
+                    - name: LDAP_REPLICATION
+                      value: "false"
+                    - name: KEEP_EXISTING_CONFIG
+                      value: "false"
+                    - name: LDAP_REMOVE_CONFIG_AFTER_SETUP
+                      value: "true"
+              volumes:
+                - name: ldap-config
+                  emptyDir: {}
+          volumeClaimTemplates:
+              - metadata:
+                  name: ldap-data
+                spec:
+                  accessModes: [ "ReadWriteOnce" ]
+                  resources:
+                    requests:
+                      storage: 10Gi
+        
+        ---
+        
+        apiVersion: v1
+        kind: Service
+        metadata:
+          labels:
+            app: phpldapadmin
+          name: phpldapadmin-service
+          namespace: kubeflow
+        spec:
+          type: ClusterIP
+          ports:
+            - port: 80
+          selector:
+            app: phpldapadmin
+        
+        ---
+        
+        apiVersion: apps/v1
+        kind: Deployment
+        metadata:
+          name: phpldapadmin
+          namespace: kubeflow
+          labels:
+            app: phpldapadmin
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              app: phpldapadmin
+          template:
+            metadata:
+              labels:
+                app: phpldapadmin
+            spec:
+              containers:
+                - name: phpldapadmin
+                  image: osixia/phpldapadmin:0.8.0
+                  ports:
+                    - name: http-server
+                      containerPort: 80
+                  env:
+                    - name: PHPLDAPADMIN_HTTPS
+                      value: "false"
+                    - name: PHPLDAPADMIN_LDAP_HOSTS
+                      value : "#PYTHON2BASH:[{'ldap-service.kubeflow.svc.cluster.local': [{'server': [{'tls': False}]},{'login': [        {'bind_id': 'cn=admin,dc=arrikto,dc=com'}]}]}]"
+        {{< /highlight >}}
+        
+        </details>
+        
+        
+    2. Seed the LDAP database with new entries.
+        
+        ```bash
+        kubectl exec -it -n kubeflow ldap-0 -- bash
+        ldapadd -x -D "cn=admin,dc=arrikto,dc=com" -W
```

Comment:
Just checking: Is the use of `arrikto` here intentional (that is, all users must use this value) or is it just an example?

The same comment applies to line 289 above.

Reply:
This is the example configuration we demoed in the community meeting.
It's an example of what an LDAP config would look like for Arrikto.
It's tested and confirmed to work.
Of course, users can substitute any values they want or use an already existing LDAP user schema.